### PR TITLE
[WIP]: linter fixes

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: src/utils/cs.ts [hunk 1]
```diff
@@ -110,17 +109,16 @@ const cs = (function() {
...
    * Filters Select to only return children with <option> tagName
    * @param {NodeList} selectList
    */
-  const filterSelectList = (selectList: NodeList): Array<Object> => {
-    return Array.from(selectList).filter((op: HTMLOptionElement) => {
-      return op.tagName === 'OPTION'; // push only <option>
-    });
-  }
+  const filterSelectList = (selectList: NodeList): Array<Object> =>
+    Array.from(selectList).filter(
```
Comment (narulakeshav) [inline src/utils/cs.ts:119]: Should this be `return Array.from(selectedList).filter(...)`?

Comment (amilajack) [reply src/utils/cs.ts:119]: I didn't make any changes to logic. These are all eslint autofixes

Comment (amilajack) [reply src/utils/cs.ts:119]: The infra wasn't configured to run against `.ts` files. Only `.tsx` files. So many files were not linted

Comment (narulakeshav) [reply src/utils/cs.ts:119]: Ahh, I totally missed that it's just returning it (no curly brace). Makes sense!

**Comment by amilajack** [comment]
[WIP] means work in progress. This wasnt supposed to be merged just yet

**Pull request merged**